profiles/apparmor.d/usr.share.openqa.script.openqa

```diff
@@ -156,6 +156,7 @@
   /usr/bin/tac mrix,
   /usr/bin/tail mrix,
   /usr/bin/wget ix,
+  /usr/bin/sed rix,
```

Comment:
```suggestion
  /usr/bin/seq rix,
```

Reply:
Ah, so the commit title is right 😄 , please also sort the lines so the sed and seq are next to each other.